apps/web/src/modules/create-proposal/components/TransactionForm/Migration/MigrationForm.tsx

```diff
@@ -0,0 +1,352 @@
+import { Box, Button, Flex, Text } from '@zoralabs/zord'
+import { Form, Formik, FormikHelpers } from 'formik'
+import { useCallback, useState } from 'react'
+import { useAccount } from 'wagmi'
+
+import DatePicker from 'src/components/Fields/Date'
+import SmartInput from 'src/components/Fields/SmartInput'
+import TextArea from 'src/components/Fields/TextArea'
+import { defaultHelperTextStyle } from 'src/components/Fields/styles.css'
+import { NUMBER, TEXT } from 'src/components/Fields/types'
+import { DropdownSelect } from 'src/modules/create-proposal'
+import { useDaoStore } from 'src/modules/dao'
+import { useLayoutStore } from 'src/stores'
+
+import { defaultInputLabelStyle } from './Migration.css'
+import migrationFormSchema, { MigrationFormValues } from './Migration.schema'
+
+export interface MigrationFormProps {
+  onSubmit?: (
+    values: MigrationFormValues,
+    actions: FormikHelpers<MigrationFormValues>
+  ) => void
+  disabled?: boolean
+  checkpoint: 0 | 1 | 2 | 3
+}
+
+export type EditionType = 'fixed' | 'open'
+
+const editionSizeOptions = [
+  { label: 'Fixed', value: 'fixed' },
+  { label: 'Open edition', value: 'open' },
+]
+
+export const MigrationForm: React.FC<MigrationFormProps> = ({
+  onSubmit,
+  disabled,
+  checkpoint,
+}) => {
+  const [editionType, setEditionType] = useState<EditionType>('fixed')
+  const [isIPFSUploading, setIsIPFSUploading] = useState(false)
+  const { address: user } = useAccount()
+  const { treasury } = useDaoStore((x) => x.addresses)
+  const isMobile = useLayoutStore((x) => x.isMobile)
+
+  const initialValues: MigrationFormValues = {
```

Comment:
guessing this was left in from copy paste is it still needed?

Reply:
looks like I actually should have deleted this file, it's not used any where